ci/helpers/hypervisor/hypervisors/kvm.py

```diff
@@ -0,0 +1,191 @@
+# Copyright 2014 iNuron NV
+#
+# Licensed under the Open vStorage Modified Apache License (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.openvstorage.org/license
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Module for the KVM hypervisor client
+"""
+
+from ci.helpers.hypervisor.apis.kvm.sdk import Sdk
+
+
+class KVM(object):
+    """
+    Represents the hypervisor client for KVM
+    """
+
+    def __init__(self, ip, username, password):
+        """
+        Initializes the object with credentials and connection information
+        """
+        _ = password
```

Comment:
I personally think using kwargs would be better when using a interface wrapping KVM and VMWARE.